### DISCUSSION TITLE
How do set the style of link underlines?

### DISCUSSION THREAD

**Discussion by allen0234** [web-infra-dev/rspress#570] [Q&A] [answered]
设置链接decoration为underline，原有的虚线下划线不消失，又多出一条下划线。怎么修改原有的这个下划线样式？
.rspress-doc a{
    color: #cb4065;
    text-decoration:underline;
}
![image](https://github.com/web-infra-dev/rspress/assets/49856281/20e2c77a-bd98-4ae8-815f-9130ba9f750e)

**Comment by sanyuan0704** [comment, marked as answer]
原有下划线是通过 border-bottom 实现的

**Reply by allen0234** [reply]
有点反直觉。我应该怎么修改这个样式？

**Reply by sanyuan0704** [reply]
加个 border-bottom: none!important; @allen0234

**Reply by allen0234** [reply]
.rspress-doc a {
    color: var(--rp-c-brand);
    text-decoration: underline;
    border-bottom: none;
}
没什么效果，原来的虚线不会消失

**Reply by sanyuan0704** [reply]
后面接个 !important 覆盖原有样式吧
